integr8ly/integreatly-operator
# MGDAPI-5308 - cleanup: removing logic for deleting cluster logging resources

# Issue link

https://issues.redhat.com/browse/MGDAPI-5308

# What

Remove logic for cleaning up cluster logging resources now that 1.34.0 has been released

# Verification steps



* Passing prow check
* Install RHOAM from this branch
```
export LOCAL=false                                                                                                            
make cluster/prepare/local
make code/run
```
* Verify RHOAM installs successfully
* Verify Red Hat OpenShift Logging can be installed successfully
  * Go to Operator Hub and install the `Red Hat OpenShift Logging` Operator to the default `openshift-logging` namespace
  * Using the UI, create the default populated `ClusterLogging` CR in the `openshift-logging` namespace
  * Using the UI, create the default populated `ClusterLogForwarder` CR in the `openshift-logging` namespace
    * Remove the forwarder output before creating to pass validation
* Verify these CRs are not deleted by the integreatly-operator
